lib/elixir/src/elixir_json.erl

```diff
@@ -0,0 +1,976 @@
+%%
+%% %CopyrightBegin%
+%%
+%% Copyright Ericsson AB 2024-2024. All Rights Reserved.
+%%
```

Comment:
I think we will have add this file to to `NOTICE`

Reply:
Done, thanks.